Library/Formula/ack.rb

```diff
@@ -1,4 +1,5 @@
 class Ack < Formula
+  desc "A search tool like grep, but optimized for programmers"
   homepage "http://beyondgrep.com/"
```

Comment:
I would prefer to keep it above the sha and version, because the sha and version apply to a single version of the software, whereas the description deals with the software regardless of version (like the homepage). Is there a reason you'd want to have it down there?